q2_diversity/tests/test_alpha_rarefaction.py

```diff
@@ -174,7 +174,7 @@ def test_invalid_invocations(self):
                 alpha_rarefaction(output_dir, t, max_depth=200,
                                   metadata=md, metrics=set(['faith_pd']))
 
-            with self.assertRaisesRegex(ValueError, 'Unknown metric: pole'):
+            with self.assertRaisesRegex(TypeError, 'pole.*incompatible'):
                 alpha_rarefaction(output_dir, t, max_depth=200,
                                   metadata=md, metrics=set(['pole-position']))
```

Comment:
![image](https://user-images.githubusercontent.com/274668/90185528-b9ffa700-dd6b-11ea-9fea-aad457fee95b.png)

Reply:
Yeah, I was glad I got to leave that in. Especially after I had to cut `unwigged_unifunc` along with `diversity_lib.test_get_nonexistent_action` this afternoon.